src/PhpSlang/Collection/ListCollection.php

```diff
@@ -15,6 +15,8 @@ class ListCollection extends AbstractCollection
      * @param array $array
      *
      * @throws ImproperCollectionInputException
+     *
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
```

Comment:
here you could iterate over `array_keys($array)` instead of supressing the warnings